README.md

```diff
@@ -86,7 +84,8 @@ After you're finished please remove all the comments and instructions!
 - **[ Has posts and users controllers ]**
 - **[ RSpecs for controler actions using request testing ]**
 - **[ Views for User and Post controllers to display content of data base to the user ]**
-
+- **[ Form to add post and comments ]**
+- **[ Contains comment controller ]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍

Reply:
@HAMMAS-SALEEM ✨ 😄  Thanks